docs/_posts/2022-05-03-42.3.5-release.md

```diff
@@ -0,0 +1,48 @@
+---
+title:  PostgreSQL JDBC Driver 42.3.5 Released
+date:   2022-05-03 13:10:32 -0400
+categories:
+    - new_release
+version: 42.3.5
+---
+**Notable changes**
+
+### Changed
+- test: polish TimestampUtilsTest
+
+### Added
+- Added KEYS file to allow for verifying artifacts [PR 2499](https://github.com/pgjdbc/pgjdbc/pull/2499)
+
+### Fixed
+- perf: enable tcpNoDelay by default [PR 2495](https://github.com/pgjdbc/pgjdbc/pull/2495).
+ This is a regression from 42.2.x versions where tcpNoDelay defaulted to true
+- docs: fix readme.md after [PR 2495](https://github.com/pgjdbc/pgjdbc/pull/2495) [PR 2496](https://github.com/pgjdbc/pgjdbc/pull/249)
+- feat: targetServerType=preferPrimary connection parameter [PR 2483](https://github.com/pgjdbc/pgjdbc/pull/2483)
+    * preferPrimary: documentation, code/logic, tests
+    * preferPrimary: simplify MultiHostChooser.candidateIterator(), fix the optimization in there
```

Comment:
I would suggest triming low-level details from the notable changes
```suggestion
```

Reply:
fair enough. I don't usually put them in.